src/ResourceManager/Sql/Commands.Sql/help/Get-AzureRmSqlDatabaseBackupLongTermRetentionPolicy.md

```diff
@@ -32,6 +32,19 @@ The policy is an Azure Backup resource used to define backup storage policy.
 
 ## PARAMETERS
 
+### -Current
+Whether or not to use the legacy Long Term Retention Vaults.```yaml
```

Comment:
So if -Current is not specified, then user gets new policy? This is a breaking change :(

Reply:
Sorry, comment is wrong. -Current gets the new, no -Current gives the legacy. I'll update the comment.